engine-rest/engine-rest-openapi/src/main/templates/paths/deployment/{id}/resources/{resourceId}/data/get.ftl

```diff
@@ -0,0 +1,55 @@
+{
+
+  <@lib.endpointInfo
+      id = "getDeploymentResourceData"
+      tag = "Deployment"
+      desc = "Retrieves the binary content of a deployment resource for the given deployment by id." />
+
+  "parameters" : [
+
+    <@lib.parameter
+        name = "id"
+        location = "path"
+        type = "string"
+        required = true
+        desc = "The id of the deployment." />
+
+    <@lib.parameter
+        name = "resourceId"
+        location = "path"
+        type = "string"
+        required = true
+        last = true
+        desc = "The id of the deployment resource." />
+
+  ],
+
+  "responses" : {
+
+    "200": {
+      "description": "Request successful. The media type of the response depends on the filename.",
+      "content": {
+        "application/octet-stream": {
+          "schema": {
+            "description": "For files without any MIME type information, a byte stream is returned."
+          }
+        },
+        "application/xml": {
+          "schema": {
+            "description": "Files with MIME type information are returned as the saved type. For example, a
```

Comment:
Here, the `media type` should be a wildcard since it depends on the MIME type. Wildcard media types are possible in the OpenAPI spec (documented here: https://swagger.io/docs/specification/media-types/)